web/template/watch.gohtml

```diff
@@ -400,13 +400,22 @@
 
         <!-- Bookmarks -->
         {{if .IndexData.TUMLiveContext.User}}
-        <div id="bookmarks-desktop" x-cloak="" x-show="showBookmarks"
-             :class="showBookmarks ? 'lg:basis-1/4' : 'lg:basis-0'"
+        <div id="bookmarks-desktop" x-cloak="" x-show="sidebar === watch.SidebarState.Bookmarks"
+             :class="sidebar === watch.SidebarState.Bookmarks ? 'lg:basis-1/4' : 'lg:basis-0'"
              class="hidden md:block basis-full h-96 lg:h-16/6 px-5 md:px-2 md:pt-2 lg:order-none order-4">
             {{template "bookmarks-modal" $stream.ID}}
         </div>
         {{end}}
 
+        <!-- Streams -->
+        {{if .IndexData.TUMLiveContext.User}}
```

Comment:
I unfortunately can't open playlists on small screens (e.g. mobile), i.e. if I press the playlist button nothing happens.